src/index.html

```diff
@@ -11,8 +11,59 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400..900&family=Roboto:
+          wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
     <h1>Product cards</h1>
+
+    <div
+      class="card card1"
+      data-qa="card"
+    >
+      <div class="card__info">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__product-code">Product code: 195434</div>
+
+      <div class="card__reviews">
+        <div class="card__review--stars stars stars--1">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__reviews--reviews">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price--how">Price</div>
+        <div class="card__price--what">$2.199</div>
```

Comment:
The price format uses a period as a thousands separator, which is unconventional in English-speaking countries where a comma is typically used. Consider changing '$2.199' to '$2,199' for consistency with common formatting standards.